queries.md

```diff
@@ -2,24 +2,73 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+select \* from Customers where City = "London";
+
 ## find all customers with postal code 1010. Returns 3 customers.
 
+select \* from Customers where PostalCode = 1010;
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
 
+select Phone from Suppliers where SupplierID = 11;
+
 ## list orders descending by the order date. The order with date 1997-02-12 should be at the top.
 
+select \* from Orders order by OrderDate desc;
+
 ## find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
 
+select \* from Suppliers where length(SupplierName) > 20;
+
 ## find all customers that include the word "market" in the name. Should return 4 records.
 
+select \* from Customers where CustomerName like '%market%';
+
 ## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
 
+insert into Customers (CustomerName, ContactName, Address, City, PostalCode, Country)
+values ('The Shire','Bilbo Baggins','1 Hobbit-Hole','Bag End','111','Middle Earth');
+
 ## update _Bilbo Baggins_ record so that the postal code changes to _"11122"_.
 
+update Customers
+set PostalCode = 11122
+where CustomerID = 92;
+
 ## list orders grouped by customer showing the number of orders per customer. _Rattlesnake Canyon Grocery_ should have 7 orders.
 
+select Customers.CustomerName as [Customer Name], Orders.\*
```

Comment:
so for this one, on your first line you really want to get the "number of orders" and not all the info -- you could implement that with a `count(orders)` type query, and since you need to reference something unique to know what to count _by_, `count(orders.orderid)` would work -- as it is now, Rattlesnake Canyon Grocery comes back with a total of 8 order instead of 7